mne/datasets/utils.py

```diff
@@ -197,27 +197,26 @@ def _get_version(name):
                          return_version=True)[1]
 
 
-def has_dataset(name):
+def has_dataset(dataset_params):
```

Comment:
@adam2392 this is an API change of a public function without deprecation.

why not allowing name to be a str or a dict and if str is passed it's understood as the key of MNE_DATASETS

so API would say simple for simplest needs

Reply:
Ah okay, wasn't sure if it required deprecation cuz it wasn't in the API docs. I'll adjust this behavior.